src/IBM.Watson.TextToSpeech.v1/ITextToSpeechService.cs

```diff
@@ -26,6 +29,7 @@ public partial interface ITextToSpeechService
         DetailedResponse<Voices> ListVoices();
         DetailedResponse<Voice> GetVoice(string voice, string customizationId = null);
         DetailedResponse<System.IO.MemoryStream> Synthesize(string text, string accept = null, string voice = null, string customizationId = null);
+        WebSocketClientTest SynthesizeUsingWebSocket(WebSocketClientTest callback, string voice = null, string customizationId = null, string accept = "");
```

Comment:
Probably wont add this to the interface - the interface will get overwritten each generation cycle. Can probably add an extension much like the extension we will add to TextToSpeechService

Reply:
Sounds good, I'm going to start cleaning up the code